manifests/m/Mozilla/Firefox/100.0/Mozilla.Firefox.installer.yaml

```
@@ -16,783 +16,7553 @@ FileExtensions:
 - htm
 - html
 - url
```

```suggestion
- avif
- pdf
- shtml
- svg
- webp
- xht
- xhtml
```
You can sort them later.